contrib/python-portend/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "python-portend"
+pkgver = "3.2.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools_scm",
+    "python-wheel",
+]
+checkdepends = ["python-pytest", "python-tempora", "python-jaraco.functools"]
```

Comment:
tempora is a direct dependency and functools is not imported by anything here

Reply:
turns out it's tempora runtime dep